src/Components/Web.JS/src/Platform/Circuits/DefaultReconnectionHandler.ts

```diff
@@ -35,11 +35,15 @@ export class DefaultReconnectionHandler implements ReconnectionHandler {
       this._currentReconnectionProcess = null;
     }
   }
+
+  onConnectionRejected(options: ReconnectionOptions) {
+    if (options.reloadOnCircuitRejected) {
+      location.reload();
+    }
+  }
 };
 
 class ReconnectionProcess {
-  static readonly MaximumFirstRetryInterval = 3000;
```

Comment:
Does anyone remember why we implemented this originally?

Reply:
Most likely because after connection goes down it would try to attempt the reconnection only after 20s, which is quite a long time.
So workaround that originally is there is to shorten the first reconnection attempt interval to 3s.